src/libs/actions/Policy.ts

```diff
@@ -1556,6 +1623,10 @@ function createWorkspaceFromIOUPayment(iouReport) {
         expenseCreatedReportActionID: workspaceChatCreatedReportActionID,
     } = ReportUtils.buildOptimisticWorkspaceChats(policyID, workspaceName);
 
+    if (!employeeAccountID || !employeeEmail) {
```

Comment:
Are you sure the function should stop execution if there's no `employeeAccountID` or `employeeEmail`?

Reply:
Yes, both are required for the `createPolicyExpenseChats` function used after this line.